spec.md

```diff
@@ -559,10 +559,52 @@ The tags MUST be in lexical order.
 
 When using the `last` query parameter, the `n` parameter is OPTIONAL.
 
+Registries MAY support an enhanced tag listing experience when clients pass an `Accept` header value of 
+`application/vnd.oci.image.index.v1+json`. If a registry supports this mode, they MUST support these additional query parameters:
+* `sort_by` - allowed values are `created_at`, `name`
+* `sort_order` - allowed values are `asc` and `desc`
+
+The response MUST be a json body in the following format:
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.index.v1+json",
+  "manifests": [
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "size": 1234,
+      "digest": "sha256:a1a1a1...",
+      "annotations": {
+        "org.opencontainers.image.ref.created": "2022-01-01T14:42:55Z",
+        "org.opencontainers.image.ref.name": "<tag1>"
```

Comment:
If a manifest contains several tags, would it be more efficient to merge all the tags into one entry here?